webapp/components/utils.js

```diff
@@ -0,0 +1,4 @@
+function ensureTrailingSlash(path) {
+  return path.endsWith('/') ? path : (path + '/');
+}
+export { ensureTrailingSlash };
```

Comment:
Missing trailing newline. (Try passing all files through git stripspace to fix this everywhere if you like.)

Reply:
```suggestion
export { ensureTrailingSlash };

```